test/common/common/logger_test.cc

```diff
@@ -0,0 +1,44 @@
+#include <string>
+
+#include "common/common/logger.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
```

Comment:
nit: add `namespace Logger` and then remove the `Logger::` qualifications below.

Reply:
Done